Bottom: 12d1d19eb521259c18612d3a7bacde5cc6af5e69
Top:    b3c51a1e90445d4437475b8cb8f9a2132832ecfb
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:58 -0500

Cleaner method to get ip address


---
diff --git a/docker/docker-exim4/docker-entrypoint.sh b/docker/docker-exim4/docker-entrypoint.sh
index c6290553..a99acea8 100755
--- a/docker/docker-exim4/docker-entrypoint.sh
+++ b/docker/docker-exim4/docker-entrypoint.sh
@@ -29,7 +29,7 @@ case ${1} in
                 && rm -rf /etc/mailname
             echo "${HOSTNAME}" > /etc/mailname
             # set local IP addr
-            LOCAL_IP_ADDR=$(grep "[ \t]*${HOSTNAME}[ \t]*" /etc/hosts|cut -f 1)
+            LOCAL_IP_ADDR=$(getent hosts $HOSTNAME | cut -d ' ' -f 1)
             sed -i 's|dc_local_interfaces=.*|dc_local_interfaces='"'${LOCAL_IP_ADDR}'"'|' ${CONFIG_PATH}/update-exim4.conf.conf
             # copy certificate
             [ -e /etc/ssl/private/exim.crt ] \
